server/services/activitiesService.ts

```diff
@@ -29,7 +29,8 @@ export default class ActivitiesService {
       // return largest possible number so that it displayed at the end of list
       if (!time) return Number.POSITIVE_INFINITY
       const [h, m] = time.split(':').map(Number)
-      return h * 60 * m
+      // convert time to minutes
+      return h * 60 + m
```

Comment:
Generally speaking, events will always have a start time. Some won't have an end time because its not mandatory in NOMIS for appointments. It is mandatory in DPS A&A. I realise this is only for sorting the rows returned, and will likely work fine, but I'll check once its in dev.